package.json

```diff
@@ -17,13 +17,14 @@
   },
   "homepage": "https://github.com/scoutforpets/objection-password#readme",
   "dependencies": {
-    "bcrypt": "^3.0.0"
+    "bcrypt": "^5.0.0"
   },
   "devDependencies": {
-    "ava": "^0.25.0",
-    "knex": "^0.14.4",
-    "objection": "^1.0.0",
-    "sqlite3": "^4.0.0",
-    "standard": "^12.0.1"
```

Comment:
I realized this after creating this PR: I removed standard because it wasn't used anywhere and I didn't know what it was, but I just looked into it and I see it's essentially an `eslint` alternative.

Should I add it back and fix any issues it finds?

Reply:
Yes, I'd prefer to keep Standard as its what I already use across the board.